client/my-sites/domains/domain-management/list/domain-row.jsx

```diff
@@ -390,9 +392,13 @@ class DomainRow extends PureComponent {
 					<PopoverMenuItem icon="info-outline" onClick={ this.goToDNSManagement }>
 						{ translate( 'Manage DNS' ) }
 					</PopoverMenuItem>
-					<PopoverMenuItem icon="book" onClick={ this.goToEditContactInfo }>
-						{ translate( 'Manage Contact Info' ) }
-					</PopoverMenuItem>
+
+					{ domain.type === domainTypes.REGISTERED &&
+						( isDomainUpdateable( domain ) || isDomainInGracePeriod( domain ) ) && (
```

Comment:
`NB:`

Just to clarify, we want contact info to be editable if a domain isn't updateable but it _is_ in a grace period?

Reply:
It has to be both:
- registered with us
- either updatable or in the grace period.